content/en-US/api/textLetterSpacing.md

```diff
@@ -37,8 +37,8 @@ canvas.width = 1000;
 canvas.height = 500;
 const ctx = canvas.getContext('2d');
 document.body.appendChild(canvas);
-const letterSpacings = [-3, 0, 3];
-ctx.font = '36px serif';
+const letterSpacings = [-2, 0, 4];
+ctx.font = '20px serif';
```

Comment:
Why this change?

Reply:
I wrote that .md file to explain how textLetterSpacing works and it's merged. I was looking at my example in chrome browser and feel that I should make it more dramatic and I reduced the font size (letters are smaller) and increased the spacing between them.